src/main.c

```diff
@@ -56,12 +53,23 @@
 // TODO: cheat on windows, cheat better?
 #define S_ISLNK S_ISREG
 #else
+#include <strings.h>
+#include <unistd.h>
+#include <sys/time.h>
 #include <editline/readline.h>
 #include <histedit.h>
 
 History *g_hist;
 #endif
 
+#ifdef _MSVC
+//#define S_ISLNK(m) (((m) & S_IFMT) == 0120000) // todo: verify correctness:
+// https://github.com/fluent/fluent-bit/blob/f07fdab96b05bba77af2da302c379c9e3f6433f4/lib/libbacktrace-ca0de05/elf.c
```

Comment:
the link reference isn't very easy to follow here, maybe change it to point to the interesting line?
the commented #define with a todo, do we need it at all? is it something we want?

Reply:
Removed the comment and created a jira ticket for it instead. HSM-294